app/org/sagebionetworks/bridge/play/controllers/StudyController.java

```diff
@@ -31,17 +33,24 @@ public int compare(Study study1, Study study2) {
             return study1.getName().compareToIgnoreCase(study2.getName());
         }
     };
-    
-    private final Set<String> studyWhitelist = Collections.unmodifiableSet(new HashSet<>(BridgeConfigFactory
-                    .getConfig().getPropertyAsList("study.whitelist")));
+
+    private final Set<String> studyWhitelist = Collections
+            .unmodifiableSet(new HashSet<>(BridgeConfigFactory.getConfig().getPropertyAsList("study.whitelist")));
 
     private UserProfileService userProfileService;
 
+    private UploadCertificateService uploadCertificateService;
+
     @Autowired
     public void setUserProfileService(UserProfileService userProfileService) {
         this.userProfileService = userProfileService;
     }
 
+    @Autowired
+    public void setUploadCertificateService(UploadCertificateService uploadCertificateService) {
```

Comment:
Minor: Mark this as final, so in case we ever subclass StudyController (unlikely), we don't accidentally override the setter and break the controller.